Projects/private-ca/playbook.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- hosts: all
+  tasks: [ ] 
+  
```

Comment:
Need to gather facts on all groups before private IPv4 addresses will be available. I was trying to pull facts on groups that were not get "fact gathered" on the play.